README.md

```diff
@@ -55,10 +55,15 @@ let
 in
 pkgs.stdenv.mkDerivation rec {
   # < ... >
+  src = ./.;
+
+  buildInputs = [ spagoPackages.installSpagoStyle ];
+
   buildPhase = 
   '' 
-    ${spagoPkgs.installSpagoStyle} # == spago2nix install
-    ${spagoPkgs.buildSpagoStyle}   # == spago2nix build
+    installSpagoStyle # == spago2nix install
+
+    ${spagoPackages.mkBuildProjectOutput { inherit src purescript; }}
```

Comment:
see below why I think `purescript` and not `purs`. Somehow it would be nice to enhance the example how to obtain this package. As `easy-purescript` is not yet in NIXOS stable, maybe a this should be somehow noted.

Reply:
yeah, really anyone who uses this tool should also be consuming easy-purescript-nix https://github.com/justinwoo/easy-purescript-nix/

Reply:
This didn't work for me, build fails with `/nix/store/whpnlfj79vd8p175rp1bwg0x7wz5lmj1-stdenv-linux/setup: line 81: /nix/store/q3m23c4r1zyvd40d5x4p2gfnmm04mdfx-build-project-output: Is a directory`